src/on-behalf-of/use-get-phone-by-account-id.tsx

```diff
@@ -0,0 +1,11 @@
+import {useGetPhoneByAccountIdQuery} from './queries/use-get-phone-by-account-id-query';
+
+export const useGetPhoneByAccountId = (accountId: string) => {
+  const {
+    data: phoneNumber,
+    isLoading,
+    isError,
+  } = useGetPhoneByAccountIdQuery(accountId);
+
+  return {phoneNumber, isLoading, isError};
+};
```

Comment:
Do we really need this layer? I think we can just skip it

Reply:
removed this layer